### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 es 4.01 (30 ottobre)

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/calcolo-delle-probabilita#309] [Esercizi M-Z]
![image](https://github.com/user-attachments/assets/c2ce7960-dde9-4f24-8575-411f0af24be9)

**Comment by erinrrr** [comment]
ho calcolato anche le "perdite", non ho finito i calcoli perché non sono sicuro sia giusto
![image](https://github.com/user-attachments/assets/f2e8da61-db97-4a8b-80d3-23f92d634ed8)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/1361e77b-456e-4e96-82d2-43067b12ce5a)
